tests/csiclone_test.go

```diff
@@ -72,4 +72,35 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component][crit:high] CSI Vol
 		verifyPVC(dataVolume, f, utils.DefaultPvcMountPath, expectedMd5)
 	})
 
+	It("[posneg:negative][test_id:6655] Support for CSI Clone strategy in storage profile with SC HPP - negative", func() {
+		if f.IsCSIVolumeCloneStorageClassAvailable() {
+			Skip("Test should only run on non-csi storage")
+		}
+
+		By(fmt.Sprintf("configure storage profile %s", cloneStorageClassName))
+		utils.ConfigureCloneStrategy(f.CrClient, f.CdiClient, cloneStorageClassName, originalProfileSpec, cdiv1.CloneStrategyCsiClone)
+
+		dataVolume, _ := createDataVolumeDontWait("dv-csi-clone-test-1", utils.DefaultImagePath, v1.PersistentVolumeFilesystem, cloneStorageClassName, f)
+		waitForDvPhase(cdiv1.CloneScheduled, dataVolume, f)
+		verifyEvent(controller.ErrUnableToClone, dataVolume.Namespace, f)
```

Comment:
I think we would want to revert the clone strategy override after we are done with the test in an AfterEach maybe?

Reply:
This is actually what is done here:
https://github.com/kubevirt/containerized-data-importer/blob/80ca2de589616e6046c224471525242eba2d7a21/tests/csiclone_test.go#L35

```
AfterEach(func() {
		if originalProfileSpec != nil {
			By("Restore the profile")
			utils.UpdateStorageProfile(f.CrClient, cloneStorageClassName, *originalProfileSpec)
		}
	})
```

Reply:
Oh missed that wasn't part of this PR.